File: KampIntro/KampIntro_Odevler/Gun_05_Odev_05/Gun_05_Odev_05/Adapters/CampaignServiceAdapter.cs
using Gun_05_Odev_05.Abstract;
using Gun_05_Odev_05.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Gun_05_Odev_05.Adapters
{
    public class CampaignServiceAdapter : ICampaignCheckService
    {
        public bool CheckIfCampaignExist(Campaign campaign)
        {
            if (campaign.CampaignId == 1)
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }
}

File: Java_React_Ödevler/Java_React_Ödev_Gün_04_Ödev_03_OyunSatis/Gun_04_Odev_03/Adapters/CampaignServiceAdapter.java

public class CampaignServiceAdapter implements CampaignCheckService {

	@Override
	public boolean CheckIfCampaignExist(Campaign campaign) {
		if (campaign.getId() == 1)
        {
            return true;
        }
        else
        {
            return false;
        }
	}

}